sdk/src/client/error.rs

```diff
@@ -160,6 +160,9 @@ pub enum Error {
     /// Missing transaction payload.
     #[error("missing transaction payload")]
     MissingTransactionPayload,
+    /// Output not unlockable due to expiration
+    #[error("output not unlockable due to expiration")]
+    OutputNotUnlockableDueToExpiration,
```

Comment:
I think this is a tiny bit confusing, one could understand that we don't own the address specified in the expiration. Maybe mention the expiration dead zone instead?

Reply:
OutputNotUnlockableDueToExpirationDeadzone? :carlos:

Reply:
Yeah bit long, maybe just `ExpirationDeadzone` + more info in the text should be fine